docs/tutorials/paroutes_benchmark.ipynb

```diff
@@ -0,0 +1,761 @@
+      "text/plain": [
+       "<rdkit.Chem.rdchem.Mol at 0x7f98a412d350>"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "target_mol = Molecule(test_smiles[100])\n",
```

Comment:
Did we ever mention that we're not actually going to run the full benchmark and rather run only on a single arbitrary molecule? I understand this is to make the notebook not take excessively long, but then maybe we should mention this to avoid confusion.

Reply:
Good point: I added a clarifying comment at the start of the tutorial.